pkg/webui/console/containers/side-bar/navigation/app-list-side-navigation.js

```diff
@@ -24,7 +24,7 @@ import SidebarContext from '../context'
 const AppListSideNavigation = () => {
   const { topEntities, isMinimized } = useContext(SidebarContext)
 
-  if (isMinimized || topEntities.length === 0) {
+  if (isMinimized) {
     return null
   }
 
```

Comment:
Yeah, it can also be put back. However, the final implementation will likely have some sort of empty-state layout where it will encourage to create an entity. Also there will be logic to fill top-entities not only with manually set bookmarks but also entities that the Console itself considers as such.